README.md

```diff
@@ -230,6 +230,12 @@ You can get the current version of the sda-cli by running:
 ./sda-cli version
 ```
 
+## Htsget
+
+You can download a (partial) file using the htsget server. The file will be downloaded in the current directory, if outdir is not defined. Filename, htsgethost, and key are required.
+```bash
+htsget -dataset <datasetID> -filename <filename> -reference <reference-number> -htsgethost <htsget-hostname> key <public-key-file> -outdir <dir>
+```
```

Comment:
It would be nice with the full example (from this PR's description) here:
`./sda-cli htsget -config testing/s3cmd.conf -dataset DATASET0001 -filename htsnexus_test_NA12878 -reference 11 -htsgethost http://localhost:8088 -key c4gh.pub.pem`
(And maybe some more of the nice instructions you added there)